### PR TITLE
Add orca tf keras quickstart & update tf quickstart

### PR DESCRIPTION
**Pull request by cyita**

### CODE REVIEW

File: docs/docs/Orca/orca-tf-quickstart-keras.md [hunk 1]
```diff
@@ -0,0 +1,100 @@
+
+**In this guide we will describe how to scale out TensorFlow (v1.15) Keras programs using Orca in 4 simple steps.**
```
Comment (jason-dai) [inline docs/docs/Orca/orca-tf-quickstart-keras.md:2]: scale out Keras (v2.3) programs

File: docs/docs/Orca/orca-tf-quickstart-keras.md [hunk 2]
```diff
@@ -0,0 +1,100 @@
...
+conda create -n zoo python=3.7 # "zoo" is conda enviroment name, you can use any name you like.
+conda activate zoo
+pip install analytics_zoo-${VERSION} # install either version 0.9 or latest nightly build
+pip install tensorflow==1.15.0
+pip install tensorflow-datasets==2.0
+pip install psutil
+pip install pandas
+pip install scikit-learn
+```
+**Note:** The original [source code](https://github.com/intel-analytics/analytics-zoo/blob/master/pyzoo/zoo/examples/orca/learn/tf/lenet/lenet_mnist_graph.py) for the tutorial below only supports TensorFlow 1.15.
```
Comment (jason-dai) [inline docs/docs/Orca/orca-tf-quickstart-keras.md:20]: source link is incorrect.

only supports TensorFlow 1.15 and Keras 2.3?

File: docs/docs/Orca/orca-tf-quickstart-keras.md [hunk 3]
```diff
@@ -0,0 +1,100 @@
...
+    init_orca_context(cluster_mode="k8s", num_nodes=2, cores=2) # run on K8s cluster
+elif args.cluster_mode == "yarn":
+    init_orca_context(cluster_mode="yarn-client", num_nodes=2, cores=2, driver_memory="6g") # run on Hadoop YARN cluster
+```
+
+This is the only place where you need to specify local or distributed mode. View [Orca Context](./context) for more details.
+
+### **Step 2: Define the Model**
+
+You may define your model, loss and metrics in the same way as in any standard (single node) TensorFlow Keras program.
```
Comment (jason-dai) [inline docs/docs/Orca/orca-tf-quickstart-keras.md:36]: standard Keras program

File: docs/docs/Orca/orca-tf-quickstart-keras.md [hunk 4]
```diff
@@ -0,0 +1,100 @@
...
+
+```python
+import tensorflow as tf
```
Comment (jason-dai) [inline docs/docs/Orca/orca-tf-quickstart-keras.md:39]: better to use `from tensorflow import keras` in the code?

File: docs/docs/Orca/orca-tf-quickstart-keras.md (hunk 2 above)
Comment (cyita) [reply docs/docs/Orca/orca-tf-quickstart-keras.md:20]: Keras v2.3 is not needed in this example. Only `Keras-application` and `Keras-preprocessing` are needed. These two packages will be installed with TensorFlow v1.15

File: docs/docs/Orca/orca-tf-quickstart-keras.md [hunk 5]
```diff
@@ -1,5 +1,5 @@
 
-**In this guide we will describe how to scale out TensorFlow (v1.15) Keras programs using Orca in 4 simple steps.**
+**In this guide we will describe how to scale out Keras programs using Orca in 4 simple steps.**
```
Comment (jason-dai) [inline docs/docs/Orca/orca-tf-quickstart-keras.md:2]: Keras (v2.3)?

File: docs/docs/Orca/orca-tf-quickstart-keras.md (hunk 2 above)
Comment (jason-dai) [reply docs/docs/Orca/orca-tf-quickstart-keras.md]: ok

File: docs/docs/Orca/orca-tf-quickstart.md [hunk 6]
```diff
@@ -98,10 +99,13 @@ Next, fit and evaluate using the Estimator.
 ```python
 est.fit(data=train_dataset,
         batch_size=320,
-        epochs=100,
-        validation_data=val_dataset)
+        epochs=5,
+        validation_data=mnist_test,
+        # tfds mnist only has one file and cannot be sharded on files,
+        # falling back on sharding on records
+        auto_shard_files=False)
```
Comment (jason-dai) [inline docs/docs/Orca/orca-tf-quickstart.md:106]: @yangw1234 Does it make sense to expose `auto_shard_files` to the user?

File: docs/docs/Orca/orca-tf-quickstart-keras.md [hunk 7]
```diff
@@ -0,0 +1,101 @@
...
+
+est = Estimator.from_keras(keras_model=model)
+```
+
+Next, fit and evaluate using the Estimator.
+```python
+est.fit(data=mnist_train,
+        batch_size=320,
+        epochs=5,
+        validation_data=mnist_test, auto_shard_files=False)
```
Comment (jason-dai) [inline docs/docs/Orca/orca-tf-quickstart-keras.md:93]: make `auto_shard_files=False` default @yangw1234 ?

File: docs/docs/Orca/orca-tf-quickstart-keras.md [hunk 8]
```diff
@@ -0,0 +1,101 @@
...
+
+result = est.evaluate(mnist_test, auto_shard_files=False)
```
Comment (jason-dai) [inline docs/docs/Orca/orca-tf-quickstart-keras.md:95]: make `auto_shard_files=False` default @yangw1234 ?

File: docs/docs/Orca/orca-tf-quickstart.md (hunk 6 above)
Comment (jason-dai) [reply docs/docs/Orca/orca-tf-quickstart.md:106]: I think we can make `auto_shard_files` false default for now

File: docs/docs/Orca/orca-tf-quickstart-keras.md [hunk 9]
```diff
@@ -0,0 +1,101 @@
+
```
Comment (jason-dai) [inline docs/docs/Orca/orca-tf-quickstart-keras.md:1]: change to `orca-keras-quickstart.md `

File: docs/docs/Orca/orca-tf-quickstart.md (hunk 6 above)
Comment (yangw1234) [reply docs/docs/Orca/orca-tf-quickstart.md:106]: Fixed by #3289

File: docs/docs/Orca/orca-tf-quickstart-keras.md (hunk 7 above)
Comment (yangw1234) [reply docs/docs/Orca/orca-tf-quickstart-keras.md:93]: Fixed by #3289

File: docs/docs/Orca/orca-tf-quickstart-keras.md (hunk 8 above)
Comment (yangw1234) [reply docs/docs/Orca/orca-tf-quickstart-keras.md:95]: Fixed by #3289

**Comment by jason-dai** [comment]
LGTM

**Comment by cyita** [comment]
Jenkins passed http://10.239.47.210:18888/job/ZOO-PR-Validation/4865/

**Pull request merged**